port/unix/omrsysinfo.c

```diff
@@ -710,6 +710,40 @@ omrsysinfo_get_processor_feature_name(struct OMRPortLibrary *portLibrary, uint32
 }
 
 
+/**
+ * Generate the corresponding string literals for the provided OMRProcessorDesc.
+ *
+ * @param[in] portLibrary The port library.
+ * @param[out] buff The cpu features output string.
+ * @param[in] BUFF_SIZE The size of the buff.
+ */
+void
+omrsysinfo_print_processor_features(struct OMRPortLibrary *portLibrary, OMRProcessorDesc *desc, char * buff, const size_t BUFF_SIZE)
```

Comment:
This looks like an identical copy. Any chance we can get away without this code duplication?

Reply:
I looked like how other APIs are implemented. 
I think we have to duplicate it for windows and unix but we can remove the implementation for common.
It seems to me that the common file (`port/common/omrsysinfo.c`) is like an interface file where we don't add the implementation.